odoo/tools/misc.py

```diff
@@ -79,12 +79,6 @@ def _exec_pipe(prog, args, env=None):
     pop = subprocess.Popen(cmd, bufsize=-1, stdin=subprocess.PIPE, stdout=subprocess.PIPE, close_fds=close_fds, env=env)
     return pop.stdin, pop.stdout
 
-def exec_command_pipe(name, *args):
```

Comment:
Doesn't seems deprecated ?

Reply:
Yep that's a mistake, I amended the commit with other changes.